lib/src/block_syntaxes/blockquote_syntax.dart

```diff
@@ -19,10 +19,15 @@ class BlockquoteSyntax extends BlockSyntax {
 
   const BlockquoteSyntax();
 
+  /// If this blockquote is ended up at a lazy continuation line.
```

Comment:
I think this should start with "Whether" [1] and maybe say "ends with a lazy continuation line." Does that sound right?

[1] https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-a-boolean-variable-or-property-comment-with-whether-followed-by-a-noun-or-gerund-phrase

Reply:
Thanks. Fixed!